motr/st/utils/helper.c

```diff
@@ -285,38 +275,22 @@ static int write_data_to_object(struct m0_obj *obj,
 				struct m0_bufvec *attr)
 {
 	int                  rc;
-	int                  op_rc;
-	int                  nr_tries = 10;
 	struct m0_op        *ops[1] = {NULL};
 
-again:
-
-	/* Create the write request */
-	m0_obj_op(obj, M0_OC_WRITE, ext, data, attr, 0, 0, &ops[0]);
-	if (ops[0] == NULL)
-		return M0_ERR(-EINVAL);
+	/* Create write operation */
```

Comment:
I don't remember the reason why we re-try to launch the write operation.
And I am not sure if that problem has been fixed.

Reply:
Exactly. So better to clean up the rubbish after ourselves. Especially, as we present this code as a reference one to the community.